internal/cmd/loadbalancer/disable_public_interface.go

```diff
@@ -14,7 +14,7 @@ import (
 var DisablePublicInterfaceCmd = base.Cmd{
 	BaseCobraCommand: func(client hcapi2.Client) *cobra.Command {
 		return &cobra.Command{
-			Use:                   "disable-public-interface [FLAGS] LOADBALANCER",
+			Use:                   "disable-public-interface [options] <load-balancer>",
```

Comment:
```suggestion
			Use:                   "disable-public-interface <load-balancer>",
```

There are no other flags.